util/dvsim/sim_utils.py

```diff
@@ -99,3 +100,135 @@ def vcs_cov_summary_table(buf):
 
     # If we reached here, then we were unable to extract the coverage.
     raise SyntaxError(f"Coverage data not found in {buf.name}!")
+
+
+def get_job_runtime(log_text: List, tool: str) -> [float, str]:
```

Comment:
```suggestion
def get_job_runtime(log_text: List, tool: str) -> Tuple[float, str]:
```

ditto on the methods below.

Reply:
sure! thanks